lib/src/components/NxNexusPageHeader/__tests__/NxNexusPageHeader.test.tsx

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
+import NxPageHeader, { HeaderLink, HeaderLinkProps } from '../NxNexusPageHeader';
+
+describe('NxPageHeader', function() {
```

Comment:
If the duplication between `NxPageHeader` and `NxNexusPageHeader` is reduced, the number of tests necessary here should be able to be greatly reduced as well.